sample_config.py

```diff
@@ -1,5 +1,7 @@
 from pytz import timezone
 
+MAINTENANCE_MODE = False
+MAINTENANCE_MODE_MESSAGE = "Maintenance will conclude at __:__ on __/__."
```

Comment:
Does this mean on-demand will automatically leave maintenance mode?
I think it might be better to have it be manual.

Reply:
Nah this is just a sample message that will be shown.